utils/src/deployer.rs

```diff
@@ -596,6 +672,19 @@ pub async fn is_proxy_contract(provider: impl Provider, addr: Address) -> Result
     Ok(impl_address != Address::default())
 }
 
+pub async fn verify_contract_address(provider: impl Provider, address: Address) -> Result<bool> {
```

Comment:
rename to `is_contract()`? 
(we probably will use `verify_contract_address()` for CREATE2 consistency check.)

Reply:
https://github.com/EspressoSystems/espresso-network/commit/6fc2d192bd2dd2c8fa15f2fc926bf9c484988ea0